database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -4,6 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 class CreateUsersTable extends Migration
+
```

Comment:
There should be no space here, it should be before the class.

Reply:
I made a mistake.
The code has been fixed.